ansible/roles/rbenv/tasks/main.yml

```diff
@@ -51,13 +51,6 @@
   become: true
   become_user: deploy
 
-- name: install ruby {{ ruby_version }}
-  shell: rbenv install -s {{ ruby_version }}
-  environment:
-    PATH: "/home/deploy/.rbenv/bin:/home/deploy/.rbenv/plugins/ruby-builds/bin:{{ ansible_env.PATH }}"
-  become: true
-  become_user: deploy
-
 - name: install ruby 2.7.4
```

Comment:
Should set `ruby_version` in ansible/group_vars/all/vars.yml and keep the var around.